easybuild/easyconfigs/m/magma/magma-1.6.2-CrayGNU-5.2.40.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "ConfigureMake"
+
+name = 'magma'
+version = '1.6.2'
+
+homepage = 'http://icl.cs.utk.edu/magma/'
+description = """The MAGMA project aims to develop a dense linear algebra library similar to LAPACK 
+    but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems."""
+
+toolchain = {'name': 'CrayGNU', 'version': '5.2.40'}
+toolchainopts = {'pic': True, 'dynamic': True,  'openmp': True, }
+
+preinstallopts = "export EBINSTALLPREFIX=%(installdir)s && "
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://icl.cs.utk.edu/projectsfiles/magma/downloads/']
+
+patches = [('make.inc', 'magma-1.6.2/')]
```

Comment:
please rename `make.inc` file to something like `magma_Cray-make.inc`

I think you can then adjust this line as:

```
patches = [('magma_Cray-make.inc', 'magma-%(version)s/make.inc')]
```